app/controllers/reservations.js

```diff
@@ -105,6 +105,54 @@ class Reservations {
 
         return res.status(404).send();
     }
+
+    static async createOrder(req, res) {
+        const model = new ReservationsModel();
+        if (!req.params.reservation_id || !model.validateId(req.params.reservation_id)) {
+            return res.status(400).send();
+        }
+
+        const result = await model.findOne();
```

Comment:
wrap this line in try catch block and inside findOne instead return error, throw error